### PR TITLE
Fix `verify_slashable_vote_data`: use `get_domain`

### PR DESCRIPTION
**Pull request by hwwhww**

### CODE REVIEW

File: specs/core/0_beacon-chain.md [hunk 1]
```diff
@@ -1023,7 +1023,11 @@ def verify_slashable_vote_data(state: BeaconState, vote_data: SlashableVoteData)
         pubkeys=pubs,
         messages=messages,
         signature=vote_data.aggregate_signature,
-        domain=DOMAIN_ATTESTATION,
+        domain=get_domain(
+            state.fork_data,
+            state.slot,
+            DOMAIN_ATTESTATION
+        ),
```
Comment (mkalinin) [inline specs/core/0_beacon-chain.md]: Looks like a redundant comma mark
```suggestion
        )
```

Comment (JustinDrake) [reply specs/core/0_beacon-chain.md]: We like redundant commas :) A comma should be added after `DOMAIN_ATTESTATION` for consistency.

Comment (mkalinin) [reply specs/core/0_beacon-chain.md]: cool 😀

Comment (hwwhww) [reply specs/core/0_beacon-chain.md]: @JustinDrake oh right. 👍 

@mkalinin we think the redundant dangling commas may reduce the size of the unrelated diff when new items are added.

e.g.,
```python
x = my_function(
    a=1,
    b=2,
    c=3
)
```

If we add a new paramaters `d`:
```python
x = my_function(
    a=1,
    b=2,
-   c=3,
+   c=3,
+   d=4
)
```

**Pull request merged**